lib/openstax/aws/deployment_base.rb

```diff
@@ -1,5 +1,24 @@
 module OpenStax::Aws
   class DeploymentBase
+    class Status
+      def initialize(deployment)
+        @deployment = deployment
+      end
+
+      def stacks
+        @deployment.stacks
+      end
+
+      def events
```

Comment:
These aren't "events", they are statuses for each stack.  But I think this is maybe the attempt at making the hash so we can write out JSON.  

Let's make methods that give us the hash and JSON for printing it out, but since we now have POROs (plain old ruby objects), let's make methods that expose those too so that people can take advantage of them:

```ruby
def stack_statuses
  @stack_statuses ||= @deployment.stacks.each_with_object({}) |stack, hash|
    hash[stack.name] = stack.status
  end
end

def to_h
  {
    stacks: stack_statuses
  }
end

def to_json
  to_h.to_json
end
```